src/Illuminate/Pipeline/Pipeline.php

```diff
@@ -104,6 +105,21 @@ public function then(Closure $destination)
     }
 
     /**
+     * Get internal container instance.
+     *
+     * @return \Illuminate\Contracts\Container\Container
+     * @throws \RuntimeException
+     */
+    protected function getContainer()
```

Comment:
How about you put a make function here instead of a getContainer function, and get it to call container make, or if there are no params to the constructor, and class_exists is true, get it to just attempt to `new` the class?